command/pr.go

```diff
@@ -357,6 +362,10 @@ func prView(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if web && !connectedToTerminal(cmd) {
+		return errors.New("--web unsupported when not attached to a tty")
```

Comment:
Could we allow `--web` in scripting scenarios? Since people can set `BROWSER` environment variable to be the tool of their choice, it might be useful to allow it. Is there some part of `--web` experience that requires an interactive terminal?

Reply:
hm; I suppose we could. I think my brain was interpreting "non-tty" as "human not present" but they might want this tool to open things in a browser as part of a larger script.

I'll undo all that here and in the other PRs.

Reply:
makes sense! My brain was accidentally superimposing "non-tty" with "no human at computer" but naturally that wouldn't be the case in a script that composes gh. I restored this behavior but did suppress the informational printing if not connected to a tty.